testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSWalletTest.scala

```diff
@@ -294,7 +294,23 @@ trait BitcoinSWalletTest extends BitcoinSFixture with EmbeddedPg {
 
   def withWalletConfig(test: OneArgAsyncTest): FutureOutcome = {
     val builder: () => Future[WalletAppConfig] = () => {
-      val walletConf = config.walletConf
+      val baseConf = config.walletConf
+      val walletNameOpt = if (NumberGenerator.bool.sampleSome) {
```

Comment:
So this will add named wallets to test cases that use fixtures right? The probability that we are using one is 1/2 ? 

If that is the case, this is nice! I always love it when our test infra get easily be modified to test a new feature :-)

Reply:
Yep!